jobs/JREGIONAL_PROCESS_BUFR

```diff
@@ -70,13 +70,28 @@ mkdir_vrfy -p ${workdir}
 #
 #-----------------------------------------------------------------------
 #
+# create COMOUT directory
+#
+#-----------------------------------------------------------------------
+
+if [ "${RUN_ENVIR}" = "nco" ]; then 
+  if [ ${CYCLE_TYPE} == "spinup" ]; then
+    COMOUT="${COMOUT_BASEDIR}/$RUN.$PDY/${cyc}_spinup${SLASH_ENSMEM_SUBDIR}"
+  else
+    COMOUT="${COMOUT_BASEDIR}/$RUN.$PDY/${cyc}${SLASH_ENSMEM_SUBDIR}"
+  fi
+  mkdir_vrfy -p "${COMOUT}"
+fi
```

Comment:
Similar to the radar data preprocessing, we don't need to have member directory since all the members use the same processed data. We just need to have this process and output data separate for the deterministic and ensemble run.

Reply:
removed